common/src/main/java/com/wynntils/mc/EventFactory.java

```diff
@@ -411,5 +412,9 @@ public static Event onSubtitleSetText(ClientboundSetSubtitleTextPacket packet) {
         return post(new SubtitleSetTextEvent(packet.getText()));
     }
 
+    public static Event onMouseScroll(double yOffset) {
+        return post(new MouseScrollEvent(yOffset > 0));
```

Comment:
Maybe send in the yOffset as is into the event, and then add a `boolean isScrollingUp() { return yOffset > 0; }` method in the event instead? That makes it a bit more general if anyone wants to use it in the future.